internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -104,7 +106,7 @@ func (r *Reconciler) reconcilePGMonitorExporter(ctx context.Context,
 	// the `EnableExporterInPostgreSQL` funcs; that way we are always running
 	// that function against an updated and running pod.
 	if pgmonitor.ExporterEnabled(cluster) {
-		sql, err := os.ReadFile(fmt.Sprintf("%s/pg%d/setup.sql", pgmonitor.GetQueriesConfigDir(), cluster.Spec.PostgresVersion))
+		sql, err := os.ReadFile(fmt.Sprintf("%s/pg%d/setup.sql", pgmonitor.GetQueriesConfigDir(log), cluster.Spec.PostgresVersion))
```

Comment:
pass the context down then get the logger from inside the `GetQueriesConfigDir` function
```suggestion
		sql, err := os.ReadFile(fmt.Sprintf("%s/pg%d/setup.sql", pgmonitor.GetQueriesConfigDir(ctx), cluster.Spec.PostgresVersion))
```

It is more normal for us to pass around a context and sets us up to do more interesting things in the future with contexts